.github/workflows/support.yml

```diff
@@ -0,0 +1,33 @@
+# Configuration for support-requests - https://github.com/dessant/support-requests
+name: 'Support Requests'
+
+on:
+  issues:
+    types: [labeled, unlabeled, reopened]
```

Comment:
~why run it when an issue is reopened?~
This workflow also removes the label when a labelled issue is reopened. I just wasn't aware this is how the bot behaved before.